src/main/java/com/puppycrawl/tools/checkstyle/Main.java

```diff
@@ -723,35 +723,34 @@ private static final class CliOptions {
 
         /** Option that controls whether to print the AST of the file. */
         @Option(names = {"-t", "--tree"},
-                description = "Prints Abstract Syntax Tree(AST) of the checked file. The option "
-                        + "cannot be used other options and requires exactly one file to run on "
-                        + "to be specified.")
+                description = "This option is used to display the Abstract Syntax Tree (AST) "
+                        + "of the specified file. It can only be used on a single file and "
+                        + "cannot be combined with other options.")
         private boolean printAst;
 
         /** Option that controls whether to print the AST of the file including comments. */
         @Option(names = {"-T", "--treeWithComments"},
-                description = "Prints Abstract Syntax Tree(AST) with comment nodes "
-                        + "of the checked file. The option cannot be used with other options "
-                        + "and requires exactly one file to run on to be specified.")
+                description = "This option is used to display the Abstract Syntax Tree (AST) "
+                        + "with comment nodes of the specified file. It can only be used on a "
+                        + "single file and cannot be combined with other options.")
         private boolean printAstWithComments;
 
         /** Option that controls whether to print the parse tree of the javadoc comment. */
         @Option(names = {"-j", "--javadocTree"},
                 description = "Prints Parse Tree of the Javadoc comment. "
```

Comment:
Follow similar pattern of others. `This option is used to ....`

In addition, make sure it is clear what is the difference between all the options.

Reply:
> Follow similar pattern of others. 

@rnveach , doing this would mean the phrase "This option is used to" repeated for 18 times. 
Please confirm if you are really okay with this duplication before we make changes.

Just a thought, how would it look if we consider organizing all the options in a tabular format (with 2 columns)?

Reply:
> doing this would mean the phrase "This option is used to" repeated for 18 times.
Please confirm if you are really okay with this duplication before we make changes.

Just pointing out that you added 3 more duplications in this PR. I am just asking for one more. :)

I am fine with this, since I am just adding 1 more.

> how would it look if we consider organizing all the options in a tabular format (with 2 columns)?

I believe this is the standard in other apache maven documentations, right?

I think this would be fine, but I this would need to be an issue first to gain approvals from the others. @nrmancuso is already trying to rewrite other parts of our documentation, so I want to sure we align with his current thoughts.

Reply:
> I am just asking for one more. :)

Omg ! My bad, I misunderstood that you were asking for uniformity in all the 18 options our Cli supports (side effects of sleep deprivation 😅 )

> I believe this is the standard in other apache maven documentations, right?

Precisely why I proposed this, yes :)
https://maven.apache.org/ref/3.6.1/maven-embedder/cli.html#

> I think this would be fine, but I this would need to be an issue first to gain approvals from the others. @nrmancuso is already trying to rewrite other parts of our documentation, so I want to sure we align with his current thoughts.

Got it..
But @rnveach, since this is a doc update, do we want a new issue or will a new discussion suffice?

Reply:
> were asking for uniformity in all the 18 options our Cli supports 

No, just the one that I highlighted.

> since this is a doc update, do we want a new issue or will a new discussion suffice?

Discussions do not gain approvals. Issues do. GH has the option to create issues from discussions.

We need issues for things that need approvals, and I think this is a big enough change we should be sure there is agreement this is what should be done. It prevents back and forth if something changes and a differing opinion comes up.

Reply:
> I think this is a big enough change we should be sure there is agreement this is what should be done.

@rnveach , Issue for further discussion created at #14759 

> No, just the one that I highlighted.

Updated 🟢